src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/ItemTests.ps1

```diff
@@ -50,32 +53,32 @@ function Test-AzureVMGetItems
 		Assert-True { $items.VirtualMachineId -contains $vm.Id }
 		Assert-NotNull $items[0].LastBackupTime
 
-		# VARIATION-3: Get items for container with ProtectionStatus filter
+		## VARIATION-3: Get items for container with ProtectionStatus filter
 		$items = Get-AzureRmRecoveryServicesBackupItem `
 			-VaultId $vault.ID `
 			-Container $container `
 			-WorkloadType AzureVM `
 			-ProtectionStatus Healthy;
 		Assert-True { $items.VirtualMachineId -contains $vm.Id }
 
-		# VARIATION-4: Get items for container with Status filter
+		## VARIATION-4: Get items for container with Status filter
 		$items = Get-AzureRmRecoveryServicesBackupItem `
 			-VaultId $vault.ID `
 			-Container $container `
 			-WorkloadType AzureVM `
 			-ProtectionState IRPending;
 		Assert-True { $items.VirtualMachineId -contains $vm.Id }
 
-		# VARIATION-5: Get items for container with friendly name and ProtectionStatus filters
-		$items = Get-AzureRmRecoveryServicesBackupItem `
-			-VaultId $vault.ID `
-			-Container $container `
-			-WorkloadType AzureVM `
-			-Name $vm.Name `
-			-ProtectionStatus Healthy;
-		Assert-True { $items.VirtualMachineId -contains $vm.Id }
+		## VARIATION-5: Get items for container with friendly name and ProtectionStatus filters
+		$items = get-azurermrecoveryservicesbackupitem `
+			-vaultid $vault.id `
+			-container $container `
+			-workloadtype azurevm `
+			-name $vm.name `
+			-protectionstatus healthy;
+		assert-true { $items.virtualmachineid -contains $vm.id }
 
-		# VARIATION-6: Get items for container with friendly name and Status filters
+		## VARIATION-6: Get items for container with friendly name and Status filters
```

Comment:
There are many unwanted changes across files. Scan all changed files and make sure only the intended changes are kept.